openapi/pagopa-arc-be.openapi.yaml

```diff
@@ -0,0 +1,112 @@
+openapi: 3.0.1
+info:
+  title: pagopa-arc-be
+  description: Api and Models
+  version: 0.0.1
+servers:
+  - url: "http://localhost:8080/arc"
+    description: Generated server url
+paths:
+  /transactions:
+    get:
+      tags:
+        - arc transactions
+      summary: "Retrieve the paged transaction list from arc"
+      operationId: getTransactionList
+      parameters:
+        - name: x-fiscal-code
+          in: header
+          required: true
+          schema:
+            type: string
```

Comment:
is this parameter not inferred from the session cache?

Reply:
yes it is. 
for the moment that we don't yet have this implementation we pass the cf directly from BE, correct?